src/utils/token.js

```diff
@@ -3,9 +3,10 @@ const config = require('../config/auth');
 
 // Generate an Access Token for the given User ID
 function generateAccessToken(userId) {
-	const expiresIn = '1 hour';
-	const { audience, issuer, secret } = config.authentication.token;
-	
+	// 1hr
+	const expiresIn = Math.floor(Date.now() / 1000 + 3600);
+	const { audience, issuer, secret } = config.token;
+
 	const token = jwt.sign({}, secret, {
```

Comment:
do we have the expiry of the token specific to application.

I guess that has to be in some common persistent cache where that has to expired by that cache not the application itself

Reply:
Ideally this token should be in some key vault , we can take all this in separate PR.